packages/core/src/components/workflow-step/workflow-step.tsx

```diff
@@ -58,7 +58,7 @@ export class WorkflowStep {
   /**
    * Activate navigation click
    */
-  @Prop() position: 'first' | 'last' | 'undefined' = 'undefined';
+  @Prop() position: 'first' | 'last' | 'single' | 'undefined' = 'undefined';
```

Comment:
I think we should annotate this prop with `@deprecated` because its more an internal prop.
```
@deprecated Will be changed to '@internal' in 2.0.0
```

Any other opionions regarding this step?

/cc @nuke-ellington